meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/notepad-app.bb

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2017 GENIVI Alliance
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+SUMMARY = "Notepad-App"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=815ca599c9df247a0c7f619bab123dad"
+SRC_URI = "git://github.com/GENIVI/notepad-app.git;protocol=https"
+SRCREV = "bba3bd458d288f43e08d526c48b823d10cb2b7d0"
+
+S = "${WORKDIR}/git"
+
+DEPENDS = "qtdeclarative persistence-client-library "
```

Comment:
Isn't keeping a space a good practice since it makes it easier to do a "_append"?